packages/editor/src/lib/editor/Editor.ts

```diff
@@ -1477,7 +1478,7 @@ export class Editor extends EventEmitter<TLEventMap> {
 		historyOptions?: TLCommandHistoryOptions
 	): this {
 		const ids = shapes.map((shape) => (typeof shape === 'string' ? shape : shape.id))
-		this._setSelectedShapes(ids, historyOptions)
+		this._setSelectedShapes(ids.sort(), historyOptions)
```

Comment:
Me and @ds300 discovered some react thrashing of selection indicator component when using brush selecting. It was caused by updating the selected shapes and not using a sorted array. Decided to add a sort here so that all the updates to selected shapes keep sort order and alleviate this. (we do something similar in `getRenderingShapes` for shapes, this does it for indicators).

Reply:
Reverted this as it broke a bunch of tests. Still might be a good idea?